controller/translationcontroller.php

```diff
@@ -95,18 +95,18 @@ public function getLanguageStrings() {
 			'revision.restored' => $this->trans->t('Revision restored'),
 
 			// js/app/controllers/settings.js
-			'bookmarklet.text' => $this->trans->t('Save in passman'),
+			'bookmarklet.text' => $this->trans->t('Save in Passman'),
```

Comment:
I am not opposed to reverting, but my issue is with this being hard to decipher for translators and users.